### DISCUSSION TITLE
收取名单里没有好友 好友统计都是空白 已经给了所有权限 最新的

### DISCUSSION THREAD

**Discussion by 13713754782** [constanline/XQuickEnergy#332] [General/我有一言当讲否]
想问问是哪里出了问题 还是我什么设置没有打开 米13 lsposed 可以偷能量 但是森林记录 庄园记录 其他记录都是空白 好友统计也没有显示好友 首页的统计也是0收能量 手机也重启过 最新的[v1.1.8-fix1]版本
![微信图片_20230905060515](https://github.com/constanline/XQuickEnergy/assets/65165892/7015d27d-e142-4e28-937a-1e06a13e754c)

**Comment by 13713754782** [comment]
忽然就全部正常显示啦

**Reply by qoocharly** [reply]
大兄弟请问最后是怎么解决的呀，我这一台redmi 12T可以，换另一台就不行了，同样是没有好友统计列表
